src/style.css

```diff
@@ -0,0 +1,117 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 430px;
+  padding: 30px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  background-image: url(/images/imac.jpeg);
+  background-size: cover;
+  background-position: center;
+  height: 134px;
+  width: 160px;
+  margin: 0 5px 45px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0 0 5px;
+}
+
+.card__code {
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0;
+  align-self: flex-start;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+  height: 15px;
+  margin: 15px 0 20px;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.stars__item {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__feedback {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  width: 100%;
+  margin: 0 0 20px;
+}
+
+.card__cost {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__sum {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__button {
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  background: #00acdc;
+  border-radius: 5px;
+  width: 170px;
+  height: 40px;
+}
+
+.card__button:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
```

Comment:
don't add border to hovered button, add it to initial state

Reply:
Done! thanks)